explorer/Dockerfile.sync

```diff
@@ -0,0 +1,11 @@
+FROM node:latest
+
+COPY package*.json ./
+
+RUN npm install
+
+COPY . .

```

Comment:
I think this isn't going to work in the CI, because the working directory of the `docker build` process is the root of repo.

Reply:
it did, on the make file theres a cd explorer.

would you prefer to change to the other way?